Release/src/pplx/pplx.cpp

```diff
@@ -27,7 +27,7 @@ namespace details
 class _Spin_lock
 {
 public:
-    _Spin_lock() : _M_lock(0) {}
+    _Spin_lock() : _M_lock() {}
```

Comment:
This compiles, but according to [this](https://en.cppreference.com/w/cpp/atomic/atomic_flag/atomic_flag) initializes the flag to an "unspecified state". Perhaps it works in all implementations we care about, but to guarantee it starts unlocked, it says we need to use `ATOMIC_FLAG_INIT` - or call `clear`.